HW5/Nazar_Savchenko/Nazar_Savchenko_05.1_Task_1.py

```diff
@@ -0,0 +1,9 @@
+# Create a list that contains elements of integer type, then use
+#the loop to change the type of these elements to a floating type.
+#(Hint: use the built-in float () function).
+num = [4, 4, 5, 4, 6, 98]
+
```

Comment:
good job
look at this version of code
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)